clh: cri-containerd: TestContainerListStatsWithSandboxIdFilter  fails

# Description of problem

Running CI with `cloud-hypervisor` fails with cri-containerd test `TestContainerListStatsWithSandboxIdFilter  `

# Actual result

```
     require.go:858: 
16:16:56         	Error Trace:	container_stats_test.go:341
16:16:56         	            				container_stats_test.go:241
16:16:56         	Error:      	Should NOT be empty, but was 0
16:16:56         	Test:       	TestContainerListStatsWithSandboxIdFilter
16:16:56 FAIL
16:16:56 ./hack/test-utils.sh: line 82: 54670 Terminated              keepalive "sudo PATH=${PATH} ${ROOT}/_output/${CONTAINERD_BIN} ${CONTAINERD_FLAGS}" ${RESTART_WAIT_PERIOD} &> ${report_dir}/containerd.log
```

The Ubuntu PR job is also failing on that test. On http://jenkins.katacontainers.io/job/kata-containers-2.0-tests-ubuntu-PR/371/consoleFull :

```
19:03:12 === RUN   TestContainerListStatsWithSandboxIdFilter
19:03:12     TestContainerListStatsWithSandboxIdFilter: container_stats_test.go:192: Create a pod config and run sandbox container
19:03:20     TestContainerListStatsWithSandboxIdFilter: container_stats_test.go:200: Create a container config and run containers in a pod
19:03:23     TestContainerListStatsWithSandboxIdFilter: container_stats_test.go:222: Fetch container stats for each container with Filter
19:03:23     TestContainerListStatsWithSandboxIdFilter: container_stats_test.go:239: Verify container stats for sandbox "0b53e74ffab32a30ec2e2482ade720d1d1fd35182d771fe6dc910ebfe748b2e4"
19:03:23     TestContainerListStatsWithSandboxIdFilter: container_stats_test.go:341: 
19:03:23         	Error Trace:	container_stats_test.go:341
19:03:23         	            				container_stats_test.go:241
19:03:23         	Error:      	Should NOT be empty, but was 0
19:03:23         	Test:       	TestContainerListStatsWithSandboxIdFilter
19:03:24 --- FAIL: TestContainerListStatsWithSandboxIdFilter (12.02s)
19:03:24 FAIL
19:03:24 ./hack/test-utils.sh: line 82:  7691 Terminated              keepalive "sudo PATH=${PATH} ${ROOT}/_output/${CONTAINERD_BIN} ${CONTAINERD_FLAGS}" ${RESTART_WAIT_PERIOD} &> ${report_dir}/containerd.log
19:03:24 Makefile:95: recipe for target 'test-integration' failed
```

--- a/integration/containerd/cri/integration-tests.sh
+++ b/integration/containerd/cri/integration-tests.sh
@@ -18,6 +18,7 @@ RUNTIME=${RUNTIME:-containerd-shim-kata-v2}
 SHIMV2_TEST=${SHIMV2_TEST:-""}
 FACTORY_TEST=${FACTORY_TEST:-""}
 KILL_VMM_TEST=${KILL_VMM_TEST:-""}
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 
 default_runtime_type="io.containerd.runtime.v1.linux"
 # Type of containerd runtime to be tested
@@ -260,7 +261,7 @@ main() {
 	TestSandboxCleanRemove
 	)
 
-	if [ "${KATA_HYPERVISOR:-}" == "cloud-hypervisor" ]; then
+	if [ "${KATA_HYPERVISOR}" == "cloud-hypervisor" ]; then
 		issue="https://github.com/kata-containers/tests/issues/2318"
 		info "${KATA_HYPERVISOR} fails with TestContainerListStatsWithSandboxIdFilter }"
 		info "see ${issue}"

--- a/integration/containerd/cri/integration-tests.sh
+++ b/integration/containerd/cri/integration-tests.sh
@@ -261,7 +261,8 @@ main() {
 	TestSandboxCleanRemove
 	)
 
-	if [ "${KATA_HYPERVISOR}" == "cloud-hypervisor" ]; then
+	if [[ "${KATA_HYPERVISOR}" == "cloud-hypervisor" || \
+		"${KATA_HYPERVISOR}" == "qemu" ]]; then
 		issue="https://github.com/kata-containers/tests/issues/2318"
 		info "${KATA_HYPERVISOR} fails with TestContainerListStatsWithSandboxIdFilter }"
 		info "see ${issue}"

